src/libraries/System.Formats.Nrbf/src/System.Formats.Nrbf.csproj

```diff
@@ -5,12 +5,13 @@
     <EnableDefaultItems>true</EnableDefaultItems>
     <CLSCompliant>false</CLSCompliant>
     <IsPackable>true</IsPackable>
+    <PackageDescription>Provides a safe reader for .NET Remoting Binary Format (NRBF) payloads.</PackageDescription>
+    <EnableDefaultPackageReadmeFile>true</EnableDefaultPackageReadmeFile>
```

Comment:
Not sure if this needed, but it's being removed when full package description added for other libraires, for example https://github.com/dotnet/runtime/pull/106785/files. I see the full package description added with https://github.com/dotnet/runtime/pull/106344/files but `EnableDefaultPackageReadmeFile` is not removed